<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PacketZoomDialog</class>
 <widget class="QDialog" name="PacketZoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>776</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>718</width>
    <height>776</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>718</width>
    <height>776</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Packet Zoom</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="labelASCII">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ASCII</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHex">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>HEX</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOpcode">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelOpcode">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Opcode :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditType">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditeSize">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelSize">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Taille :</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserASCII">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>331</width>
     <height>301</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserHex">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>331</width>
     <height>301</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReloadScript">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>730</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Recharger le script</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadScript">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>730</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Charger un script</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidgetPacket">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>691</width>
     <height>321</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>Cl√©</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valeur</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelUsedScript">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chemin du script :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNotReadByte">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>370</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Byte non lus : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
